roles/StackStorm.st2web/tasks/main.yml

```diff
@@ -61,6 +61,17 @@
   notify:
     - restart nginx
   tags: st2web
+  when: not st2web_nginx_config
+
+- name: Copy custom Nginx config
```

Comment:
Would be good then to modify previous section to: `name: Copy default Nginx config`